docs/usage/api.md

```diff
@@ -4,6 +4,8 @@ title: API
 description: How to use the Node.js API.
 permalink: /docs/usage/api/
 package: babel-core
```

Comment:
I moved this doc in babel/babel (PR babel/babel#4938). It will be then included in the website at build-time.

Could you apply your modifications in babel repo ? Thanks.